cpp/velox/compute/VeloxBackend.cc

```diff
@@ -113,21 +113,21 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
   google::InitGoogleLogging("gluten");
 
   // Allow growing buffer in another task through its memory pool.
-  config::globalConfig.memoryPoolCapacityTransferAcrossTasks = true;
+  FLAGS_velox_memory_pool_capacity_transfer_across_tasks = true;
 
   // Avoid creating too many shared leaf pools.
-  config::globalConfig.memoryNumSharedLeafPools = 0;
+  FLAGS_velox_memory_num_shared_leaf_pools = 0;
 
   // Set velox_exception_user_stacktrace_enabled.
-  config::globalConfig.exceptionUserStacktraceEnabled =
+  FLAGS_velox_exception_user_stacktrace_enabled =
       backendConf_->get<bool>(kEnableUserExceptionStacktrace, kEnableUserExceptionStacktraceDefault);
 
   // Set velox_exception_system_stacktrace_enabled.
-  config::globalConfig.exceptionSystemStacktraceEnabled =
+  FLAGS_velox_exception_system_stacktrace_enabled =
       backendConf_->get<bool>(kEnableSystemExceptionStacktrace, kEnableSystemExceptionStacktraceDefault);
 
   // Set velox_memory_use_hugepages.
-  config::globalConfig.memoryUseHugepages = backendConf_->get<bool>(kMemoryUseHugePages, kMemoryUseHugePagesDefault);
+  FLAGS_velox_memory_use_hugepages = backendConf_->get<bool>(kMemoryUseHugePages, kMemoryUseHugePagesDefault);
```

Comment:
Hi @PHILO-HE 

Thank you for working on the update.

I would suggest also removing the `DECLARE_*` macros for these Velox build-in flags from this file so we can get a compilation error once Velox removes one.

Reply:
@zhztheplayer, seems we have to keep the declaration. Otherwise, an error will be reported: xxx was not declared in this scope. If Velox removes them (removing their definition), there will be a link error reported.

Reply:
> Otherwise, an error will be reported: xxx was not declared in this scope.

That may mean some Velox headers that declared the flags are not included. Sure, it sounds valid since we still get a link error anyway.